src/WebJobs.Script.WebHost/Standby/StandbyManager.cs

```diff
@@ -102,12 +105,22 @@ public async Task SpecializeHostCoreAsync()
             // user dependencies
```

Comment:
We should add a metric for all of Specialization (I can't comment directly on it, but in this code -- https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script.WebHost/Standby/StandbyManager.cs#L69-L81):

Reply:
I added a metric in PlaceholderSpecializationMiddleware that covers this. Did you mean to say that this needs to be more granular and SpecializeHostAsync() needs to have a separate metric?

Reply:
Yeah -- specialization can happen in two ways:
1. Middleware
2. Specialization Timer that checks for specialization every 500ms

So if the middleware isn't the one that triggers it, it may get odd timing results. No reason we can't check it in both places, though.